contents/teams/replay/objectives.mdx

```diff
@@ -1,24 +1,18 @@
-### 🎯 Goal 1: Quality and Consolidation for Session Replay (web & mobile)
+### 🎯 Goal 1: People think of PostHog as a mobile solution
 
-Why? 
+* Mobile replay for React Native
+* Lots of content on how to use PostHog on Mobile
 
-We've done a lot of investigation and preparation work and we want to capitalize on that
+### 🎯 Goal 2: Error tracking in people's hands
```

Comment:
Wonder if we should be somewhat more ambitious with what we target. Given we're off to a good start I think we should aim for a concrete state. "In beta" or "Ready for public release" don't seem wild

Reply:
```suggestion
### 🎯 Goal 2: Error tracking in beta
```